### ISSUE TITLE
ATR stoploss adjustment improvement

### ISSUE THREAD

**Issue by MrGr33n47** [EarnForex/PositionSizer#43]
Currently when you want a stop loss set automatically by a ATR multiple it adds or substracts the distance from the entry level which makes sense and now problem here. But what about those who wants a ATR stop based on a near by swing? Why not add an extra line that the user can move towards the swing/support/resistance level that he/she wants the ATR stop value added or substracted from? So maybe create an option where if placed to true the user can manually set where he/she wants the ATR stop value added/substracted from if he/she feels they don't want to make use of the entry level as a level to adjust ATR stop from, but rather place the stop loss automatically a distance from the support/resistance level manually adjusted by the user. It would be a great add of a function and solve of the problem having to only stick to the entry level as a level the ATR is deducted/added from/to. Another bonus would be if the spread is also taking in consideration when calculating the ATR stop when you are selling a position. By this I mean: stoploss = (atr * multiplier) + entry level/support/resistance level + spread.. When entering a sell we exit by the ask price line not the bid price line, so in currency pairs with larger spread it makes better sense to take in consideration the adding of the spread to the sell stoploss to cater for the distance of spread as the price approaches your sell stoploss. For a buy position its not needed cause a buy position is exited by bid price. This will be more noticeable for intra-day traders and scalpers that has to deal with some spreads that is not that in their favour for a while. Please consider thank you.

**Comment by onedao** [comment]
Best Regards ,Jianhua ZhuangXishi Co.,Ltd.+86 188 7652 3758  Phone+86 0595 8600 1500 Telhttp://www.xishigroup.com

**Comment by EarnForex** [comment]
Hello!

I am already considering the spread adjustment feature for ATR multipliers. I hope it will be released in the next version of the calculator.

As for the ATR-from-arbitrary-level, I am not so sure.

**Comment by MrGr33n47** [comment]
thank you, I understand, I might attempt to add it manually in the code with the version I have for my own convenience.

**Comment by EarnForex** [comment]
Added in 3.00.
